### PR TITLE
Linux bridge

### PR DESCRIPTION
**Pull request by kobi86**
@phoracek @RamLavi 

Start working on new feature, Linux bridge

### CODE REVIEW

File: setup-ovs.sh [hunk 1]
```diff
@@ -45,8 +45,7 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
                  conn.interface brcnv master brcnv-port \
                  ipv4.method auto \
                  ipv4.dhcp-client-id "mac" \
-                 connection.autoconnect no \
-                 802-3-ethernet.cloned-mac-address $mac
+                 connection.autoconnect no
```
Comment (RamLavi) [inline setup-ovs.sh]: I'm confused, don't we want to remove this script in this branch?

Comment (kobi86) [reply setup-ovs.sh]: Yes we did :) 
But I don't have permissions to delete files in the repo 

Screen Shot 2021-04-27 at 16 53 25

Comment (phoracek) [reply setup-ovs.sh]: Sure you do - push a commit removing them

Comment (kobi86) [reply setup-ovs.sh]: I pushed the comment with git rm to the file and I get this message that I don't have permissions

Comment (phoracek) [reply setup-ovs.sh]: Could you just delete those files and do `git add`?

Comment (kobi86) [reply setup-ovs.sh]: This is what I did, have a look at this: 
https://github.com/kobi86/rhcos-slb/tree/linux-bridge

Comment (phoracek) [reply setup-ovs.sh]: Just to be clear, you did not `git rm` there?

Comment (kobi86) [reply setup-ovs.sh]: What? this is the repo from where I am creating the PR...

Comment (phoracek) [reply setup-ovs.sh]: Could you try rebasing your repo/branch on the current HEAD of this repo? I think that may be all that is missing.

The note about "write access" is only related to the fact that you can't click the merge button AFAIK.

**Pull request merged**